regulations/tests/html_builder_test.py

```diff
@@ -312,7 +312,11 @@ def test_human_label(self):
         self.assertEqual(
             'FR #111_22',
             PreambleHTMLBuilder.human_label({'label': ['111_22']}))
+        node = {
+            'label': ['111_22', 'III', 'A', 'ii', '4', 'p3', 'p7'],
+            'indexes': [('p3', 2), ('p7', 4)],
```

Comment:
Double sorry--it would include all the labels but the first, but only the paragraph labels are going to be checked in `human_label`. If we don't change the code under test, I'll revise to pass in realistic values.